flixel/ui/FlxButton.hx

```diff
@@ -312,6 +312,8 @@ class FlxTypedButton<T:FlxSprite> extends FlxSprite implements IFlxInput
 		
 		if (_spriteLabel != null && _spriteLabel.visible)
 		{
+			_spriteLabel.scale.set(scale.x, scale.y);
```

Comment:
I'm not sure this is a good idea... it breaks all code that sets the `scale` of the label directly, since those values will just always be overriden by the button's `scale` now.